STIR is incompatible SWIG 4.4

```
C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(8451,16): error C2039: 'stop_iteration': is not a member of 'swig' [C:\projects\stir\build\src\swig\_stir.vcxproj]
      C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(8311,17):
      see declaration of 'swig'
      C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(8451,16):
      the template instantiation context (the oldest one first) is
          C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(9204,24):
          see reference to function template instantiation 'swig::SwigPyIterator *swigstir::make_forward_iterator<stir::FullArrayIterator<stir::Array<3,elemT> *,stir::FullArrayIterator<stir::Array<2,elemT> *,stir::FullArrayIterator<stir::Array<1,elemT> *,float *,elemT,float &,float *>,elemT,float &,float *>,elemT,float &,float *>>(const OutIter &,const OutIter &,const OutIter &,PyObject *)' being compiled
          with
          [
              elemT=float,
              OutIter=stir::FullArrayIterator<stir::Array<3,float> *,stir::FullArrayIterator<stir::Array<2,float> *,stir::FullArrayIterator<stir::Array<1,float> *,float *,float,float &,float *>,float,float &,float *>,float,float &,float *>
          ]
          C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(8468,16):
          see reference to class template instantiation 'swigstir::SwigPyForwardIteratorClosed_T<OutIter,float,swig::from_oper<ValueType>>' being compiled
          with
          [
              OutIter=stir::FullArrayIterator<stir::Array<3,float> *,stir::FullArrayIterator<stir::Array<2,float> *,stir::FullArrayIterator<stir::Array<1,float> *,float *,float,float &,float *>,float,float &,float *>,float,float &,float *>,
              ValueType=float
          ]
          C:\projects\stir\build\src\swig\CMakeFiles\_stir.dir\stirPYTHON_wrap.cxx(8447,27):
          while compiling class template member function 'swig::SwigPyIterator *swigstir::SwigPyForwardIteratorClosed_T<OutIter,float,swig::from_oper<ValueType>>::incr(size_t)'
          with
          [
              OutIter=stir::FullArrayIterator<stir::Array<3,float> *,stir::FullArrayIterator<stir::Array<2,float> *,stir::FullArrayIterator<stir::Array<1,float> *,float *,float,float &,float *>,float,float &,float *>,float,float &,float *>,
              ValueType=float
          ]
```
This is in lines like this
https://github.com/UCL/STIR/blob/c6ee75abdf2f13a7c2af3950500ede70f0725b84/src/swig/stir.i#L765

--- a/src/swig/stir.i
+++ b/src/swig/stir.i
@@ -728,6 +728,8 @@ namespace std {
 %{
   namespace swigstir {
 #ifdef SWIGPYTHON
+
+#if SWIG_VERSION < 0x040000
   template<typename OutIterator, 
 	   typename ValueType = typename std::iterator_traits<OutIterator>::value_type,
     typename FromOper = swig::from_oper<ValueType> >
@@ -781,6 +783,18 @@ namespace std {
   {
     return new SwigPyForwardIteratorClosed_T<OutIter>(current, begin, end, seq);
   }
+
+#else
+
+  template<typename OutIter>
+  inline swig::SwigPyIterator*
+  make_forward_iterator(const OutIter& current, const OutIter& begin,const OutIter& end, PyObject *seq = 0)
+  {
+    return new swig::SwigPyForwardIteratorClosed_T<OutIter>(current, begin, end, seq);
+  }
+
+#endif // SWIG pre 4.0
+
 
 #endif
 

